config/database.yml

```diff
@@ -13,5 +13,5 @@ development:
   database: c2_development
 test:
   <<: *default
-  database: c2_test
+  url: postgresql://localhost/c2_test
```

Comment:
I was referencing http://edgeguides.rubyonrails.org/configuring.html#connection-preference

What I've seen before is a TEST_DATABASE_URL env var supported in the `database.yml` explicitly, so that if you want to override the default in `.env` you can, but with a different var explicitly for automated tests.

I may do that instead, and add comments.